docs/_index.yaml

```diff
@@ -8,22 +8,36 @@ landing_page:
     items:
     - classname: devsite-landing-row-50
       description: >
-        The computations you'll use TensorFlow for—like training a massive deep
-        neural network—can be complex and confusing. To make it easier to
-        understand, debug, and optimize TensorFlow programs, we've included a
-        suite of visualization tools called TensorBoard.
+        TensorBoard provides the visualization and tooling needed for 
+        machine learning experimentation. It enables:
+        <ul style="padding-left: 20px; margin: 20px 0;">
+          <li>Tracking and visualizing metrics such as loss and accuracy</li>
+          <li>Visualizing the model graph (ops and layers)</li>
+          <li>Viewing histograms and distributions of Tensors (e.g. weights 
+          and biases) to track how they change over time</li>
+          <li>Projecting embeddings to a lower dimensional space</li>
+          <li>Displaying images, text, and audio data</li>
+          <li>Profiling TensorFlow programs</li>
+          <li>And much more</li>
+        </ul>
 
-      image_path: /images/graph_vis_animation.gif
+      image_path: /images/tensorboard.gif
```

Comment:
We use `/images/*` here but `/resources/images/*` below; is this right?
(I see the `/resources/images/*` files within google3 but I don’t see
where they come from or how our `./images` might be merged into the same
path.)

Reply:
I am not sure. I think the references to `/resources/images/*` were in the original addition from  @lamberta. I'm happy to move the GIF to another folder if it makes sense. Do the other images need to be moved to this repo or would they be added as part of the website publishing?

Reply:
For images you want to keep within your site, creating a local `./images` directory is fine and use a relative path to include it.
For large files, we need to put them in our main site images directory in g3. We can create an `$SITE_ROOT/images/tensorboard/...` directory for you

Reply:
Per offline conversation, leaving the resource card image as is for now. We'll update this a bit later.